boa/src/exec/mod.rs

```diff
@@ -670,6 +605,46 @@ impl Interpreter {
         &self.realm
     }
 
+    /// Utility to create a function Value for Function Declarations, Arrow Functions or Function Expressions
+    pub(crate) fn create_function(
+        &mut self,
+        args: &Box<[FormalParameter]>,
```

Comment:
So, we have two options here. Either we receive a `Box<[FormalParameter]>` or a `&[FormalParameter]`. If you take a `Box`, you can directly pass it to `FunctionObject::create_ordinary` and this would probably reflect better how the function works internally, but you would need to clone it on call.

If this is not possible, you can just receive a `&[FormalParameter]` and then do a `to_vec().into_boxed_slice()`, or something like that.

Reply:
Thanks, will try both options later